tests/baselines/reference/targetTypeArgs.types

```diff
@@ -37,7 +37,7 @@ foo(function(x) { x });
 >["hello"] : string[]
 >"hello" : "hello"
 >every : { <S extends string>(predicate: (value: string, index: number, array: string[]) => value is S, thisArg?: any): this is S[]; (predicate: (value: string, index: number, array: string[]) => unknown, thisArg?: any): boolean; }
->function(v,i,a) {return true;} : (v: string, i: number, a: string[]) => true
+>function(v,i,a) {return true;} : (v: string, i: number, a: string[]) => boolean

```

Comment:
This is one of those funky places where parameter type fixing would make the return type's type contextually type itself in overload passes after the first one. That no longer occurs.